hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -353,7 +357,14 @@ public Response put(
       throw ex;
     } finally {
       if (output != null) {
-        output.close();
+        try {
+          output.close();
+        } catch (IllegalStateException ex) {
+          LOG.error(String.format(
```

Comment:
This can occur due to client behavior and we should not be logging at level `error` a client with a bad connection could fill up the S3G logs.

Reply:
The metric at line 320 should be moved below into the `finally` block.